modules/nw-router-configuring-dual-stack.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * networking/routes/route-configuration.adoc
+
+[id="nw-router-configuring-dual-stack_{context}"]
+= Configuring the {product-title} Ingress Controller for dual-stack support
```

Comment:
bit of a nit, but "dual-stack support" might not have the info payload that one of the foll might have:

"...for dual-stack networking"
"...for IPv4 and IPv6 networking"

Or keep what you have.  Up to you.